bokeh/server/views/root_handler.py

```diff
@@ -63,11 +63,11 @@ async def get(self, *args, **kwargs):
             index = "app_index.html" if self.index is None else self.index
             self.render(index, prefix=prefix, items=sorted(self.applications.keys()))
 
-    # NOTE: The methods below exist on both AuthMixin and RequestHandler. This
+    # NOTE: The methods below exist on both BokehAuthRequestHandler and RequestHandler. This
     # makes it explicit which of the versions is intended to be called.
-    get_login_url = AuthMixin.get_login_url
-    get_current_user = AuthMixin.get_current_user
-    prepare = AuthMixin.prepare
+    get_login_url = BokehAuthRequestHandler.get_login_url
+    get_current_user = BokehAuthRequestHandler.get_current_user
+    prepare = BokehAuthRequestHandler.prepare
```

Comment:
This comment and these assignments should no longer necessary (the main reason to switch to single inheritance was precisely to be able to get rid of this clunkiness)